src/transformers/pipelines/image_feature_extraction.py

```diff
@@ -0,0 +1,92 @@
+from typing import Dict
```

Comment:
we need a licence

Reply:
I'll add one 👍  The reason there isn't one here is that there isn't one for all of the `pipelines/xxx.py` files.